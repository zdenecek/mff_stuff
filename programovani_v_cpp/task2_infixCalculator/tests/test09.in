5
E=1
$e
$e+$E
$e=2
$e