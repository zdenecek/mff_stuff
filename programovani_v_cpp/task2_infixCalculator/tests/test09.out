1
1
2
invalid
1
